eng/clipack/Common.projitems

```diff
@@ -0,0 +1,40 @@
+<Project>
+  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
```

Comment:
I would have expected using the NoBuild targets sdk instead. Is there a reason why we need this one? Without it we can probably avoid having to set a target Framework.